guide/statements.md

```diff
@@ -148,3 +148,58 @@ fn bar() {
     foo();
 }
 ```
+
+### Let else statements
+
+The `else` should have once space before and after. It can also be at the beginning of a line with a space after.
+
+If the `else` block only contains only an expression, the entire statement should be on one line if it fits.
+
+```rust
+let Some(1) = opt else { return };
+```
+
+If the line needs to be broken, prefer to break after the opening brace of the `else` block
+
+```rust
+let Some(1) = opt else {
+    return
+};
+```
+
+If the `else` and opening brace do not fit on the same line as the initializer,
+break before `else`. If a line begins with `else`, it should be indented at the same level as `let`,
+and the block should be on the same line if it fits.
```

Comment:
```suggestion
and there should always be a newline after the opening brace so that the block always appears on a subsequent line. This calls attention to the `else` block attached to the `let`, making it easy to visually distinguish a `let`-`else` from a plain `let`.
```

Reply:
> This calls attention to the `else` block

The style guide does not need to say "why" right?

I think what you are suggesting here should not be a dependent clause of "if a line begins with `else`". How about this?

> If the `else` is not on the same line as `let`, the `else` block should not be written in one line.

I'm assuming that we can lean on the "blocks" section of the guide and so "not written in one line" is specific enough.

Regardless of how it's worded, I agree, this is more consistent with if/else.